plugins/git/git_commit_behind

```diff
@@ -0,0 +1,283 @@
+#! /usr/bin/env python3
+
+"""=cut
+=head1 NAME
+
+git_commit_behind - Munin plugin to monitor local git repositories and report
+how many commits behind their remote they are
+
+=head1 NOTES
+
+This plugin is similar to how apt_all works for apt packages.
+
+To be able to check how behind a git repository is, we need to run git fetch.
+To avoid fetching all repos every 5 minutes and slowing down the munin-node,
```

Comment:
Please update with regards to the optional nature of the cron job. How about the following?
> To avoid fetching all repos every 5 minutes (for each munin period) and thus slowing down the data collection, the `git fetch` operation is only randomly triggered (based on `env.update.probability`). In case of very time-consuming update operations, you can run them in a separate cron job.